include/avif/avif.h

```diff
@@ -150,7 +150,7 @@ typedef enum avifResult
     AVIF_RESULT_ENCODE_COLOR_FAILED,
     AVIF_RESULT_ENCODE_ALPHA_FAILED,
     AVIF_RESULT_BMFF_PARSE_FAILED,
-    AVIF_RESULT_NO_AV1_ITEMS_FOUND,
+    AVIF_RESULT_NO_AV1_ITEMS_FOUND, // TODO(yguyon): Rename or add AVIF_RESULT_NO_AV2_ITEMS_FOUND
```

Comment:
It seems that this error code is used to report that the primary item is missing, or the primary item or the alpha auxiliary item (non-grid) has a zero size. Whe we report this error, we haven't called `avifDecoderItemValidateAV1` (replaced by `avifDecoderItemValidateProperties` in the new code). So it seems better to rename this error code to be more generic. We can keep the old enum for backward compatibility.

When we do that, we need to explicitly assign values to the enum constants, which is a good thing to do in general when an enum type has many constants.

Reply:
Done in https://github.com/AOMediaCodec/libavif/pull/1394.